.docker/php/80-fpm-dev/Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM docker.pkg.github.com/fogospt/fogosapi/php:8.0-fpm-prod
+FROM ghcr.io/fogospt/fogosapi-php:8.0-fpm-prod
 
 LABEL description="Fogos.pt API Development PHP 8.0 image"
+LABEL org.opencontainers.image.source https://github.com/FogosPT/fogosapi
+ENV COMPOSER_HOME /tmp
 
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
Can you help me understand why this is important to be added? I literally never used it on any image I've built in the past, but I might be missing something...

Reply:
This was added as a hint from [hadolint)(https://github.com/hadolint/hadolint) , the rule [DL4006](https://github.com/hadolint/hadolint/wiki/DL4006)
The rational makes sense to me, if you want we can discuss about that

Reply:
Ah, got it! Let it be there, then.